test/test_prototype_transforms_functional.py

```diff
@@ -1185,15 +1185,15 @@ def test_correctness_gaussian_blur_image_tensor(device, spatial_size, dt, ksize,
     torch.testing.assert_close(out, true_out, rtol=0.0, atol=1.0, msg=f"{ksize}, {sigma}")
 
 
+# TODO: I guess we need to change the name of this test. Should we have a
+# _correctness test as well like the rest?
```

Comment:
We can, but I don't think there is a reason to keep this standalone anymore. Since you haven't looked at our testing architecture yet, let's keep this in backlog until then.

Reply:
As discussed offline, I'll send a patch for this after the PR.